components/Workspaces/RepositoryStatCard.tsx

```diff
@@ -123,8 +143,14 @@ export const RepositoryStatCard = ({ stats, type, isLoading, hasError }: Reposit
               .map(([stat, value]) => {
                 return (
                   <tr key={stat} className="flex flex-col">
-                    <th scope="row" className="capitalize font-normal text-lg lg:text-sm text-light-slate-12 text-left">
-                      {stat.replace("_", " ")}
+                    <th scope="row" className="capitalize font-normal text-xs lg:text-sm text-light-slate-12 text-left">
+                      {type === "stars" || type === "forks"
+                        ? stat === "over_range"
+                          ? `Over ${stats.range} Days`
+                          : stat === "average_over_range"
+                          ? `Avg. Over ${stats.range} Days`
```

Comment:
```suggestion
                          ? `Avg. per day`
```

This is confusing "Avg. over 30 days." It'd make more send to stat "Average per day."